internal/ingress/annotations/annotations.go

```diff
@@ -49,6 +49,7 @@ const (
 	RegexPriorityKey     = "/regex-priority"
 	HostHeaderKey        = "/host-header"
 	MethodsKey           = "/methods"
+	SNIKey               = "/snis"
```

Comment:
optional uber-nit: `SNIsKey` - before this change, names consistently pluralize const names to match plurality of the literal, this change breaks the rule